pkg/contract_watcher/shared/retriever/address_retriever_test.go

```diff
@@ -82,8 +82,9 @@ var _ = Describe("Address Retriever Test", func() {
 
 	Describe("RetrieveTokenHolderAddresses", func() {
 		It("Retrieves a list of token holder addresses from persisted event logs", func() {
-			addresses, err = r.RetrieveTokenHolderAddresses(*info)
-			Expect(err).ToNot(HaveOccurred())
+			var retrieveErr error
+			addresses, retrieveErr = r.RetrieveTokenHolderAddresses(*info)
```

Comment:
super tiny, but I think we could probably change this to `addresses, retrieveErr := r.RetrieveTokenHolderAddresses(*info)` instead of defining `retrieveErr ` on the line above.